src/Css.rei

```diff
@@ -667,6 +667,8 @@ let flex3:
 let flexBasis: [ Types.FlexBasis.t | Types.Length.t] => rule;
 let gridAutoColumns: [ Types.Length.t | `auto] => rule;
 let gridAutoRows: [ Types.Length.t | `auto] => rule;
+let gridArea: string => rule;
```

Comment:
For grid area, given its definition in https://developer.mozilla.org/fr/docs/Web/CSS/grid-area, I think something like this is more accurate:
```
module GridArea: {
  type t = [ `auto | `ident(string) | `num(int) | `numIdent(int, string) | `span([ | `num(int) | `ident(string) ])];
}

gridArea:  [ GridArea.t | Cascading.t ] => rule
gridArea2: (GridArea.t, GridArea.t) => rule;
gridArea3: (GridArea.t, GridArea.t, GridArea.t) => rule;
gridArea4: (GridArea.t, GridArea.t, GridArea.t, GridArea.t) => rule;
```

Reply:
I honestly didn't even know the `/` syntax for grid-area was a thing! :astonished: 

Those make sense though, thank you will add!